apm-sniffer/config/agent.config

```diff
@@ -320,3 +320,17 @@ plugin.nettyhttp.supported_content_types_prefix=${SW_PLUGIN_NETTYHTTP_SUPPORTED_
 plugin.rocketmqclient.collect_message_keys=${SW_PLUGIN_ROCKETMQCLIENT_COLLECT_MESSAGE_KEYS:false}
 # If set to true, the tags of messages would be collected by the plugin for RocketMQ Java client.
 plugin.rocketmqclient.collect_message_tags=${SW_PLUGIN_ROCKETMQCLIENT_COLLECT_MESSAGE_TAGS:false}
+# Define the max length of collected HTTP parameters. The default value(=0) means not collecting.
+plugin.solon.http_params_length_threshold=${SW_PLUGIN_SOLON_HTTP_PARAMS_LENGTH_THRESHOLD:0}
+# Define the max length of collected HTTP header. The default value(=0) means not collecting.
+plugin.solon.http_headers_length_threshold=${SW_PLUGIN_SOLON_HTTP_HEADERS_LENGTH_THRESHOLD:0}
+# It controls what header data should be collected, values must be in lower case, if empty, collect all. default is empty.
+plugin.solon.include_http_headers=${SW_PLUGIN_SOLON_INCLUDE_HTTP_HEADERS:}
+# Define the max length of collected HTTP body. The default value(=0) means not collecting.
+plugin.solon.http_body_length_threshold=${SW_PLUGIN_SOLON_HTTP_BODY_LENGTH_THRESHOLD:0}
+# Intercept class name, default is org.noear.solon.core.mvc.ActionDefault
+plugin.solon.intercept_class_name=${SW_PLUGIN_SOLON_INTERCEPT_CLASS_NAME:org.noear.solon.core.mvc.ActionDefault}
+# Intercept method name, default is invoke
+plugin.solon.intercept_method_name=${SW_PLUGIN_SOLON_INTERCEPT_METHOD_NAME:invoke}
```

Comment:
Inceptor method should not be defined by the configurations. This is a kind of config abuse.